vnext/Directory.Build.props

```diff
@@ -17,10 +17,9 @@
 
   <PropertyGroup Label="Configuration">
     <ProjectName Condition="'$(ProjectName)'==''">$(MSBuildProjectName)</ProjectName>
-
     <!-- Visual Studio forces using 'Win32' for the 'x86' platform. -->
-    <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(ReactNativeWindowsDir)build\$(Platform)\$(Configuration)</BaseIntDir>
-    <BaseIntDir Condition="'$(Platform)' == 'Win32'">$(ReactNativeWindowsDir)build\x86\$(Configuration)</BaseIntDir>
+    <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(ReactNativeWindowsDir)\$(Platform)\$(Configuration)</BaseIntDir>
```

Comment:
Please revert. This will place intermediate build artifacts under, for instance, `C:\repos\rnw\vnext\x64` or `C:\repos\rnw\vnext\ARM`.

The default location should be centralized: `C:\repos\rnw\vnext\build\[ARM|x86|x64]...`

Reply:
```suggestion
    <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(ReactNativeWindowsDir)build\$(Platform)\$(Configuration)</BaseIntDir>
```